sdk/python/kfp/_client.py

```diff
@@ -1010,20 +1032,60 @@ def list_recurring_runs(self, page_token='', page_size=10, sort_by='', experimen
       response = self._job_api.list_jobs(page_token=page_token, page_size=page_size, sort_by=sort_by)
     return response
 
-  def get_recurring_run(self, job_id):
+  def get_recurring_run(self, job_id: str = None, job_name: str = None, namespace: str = None):
```

Comment:
It's a pity the KFP SDK codebase doesn't have type annotations for outputs. I was going to ask you to write the output annotation, but we can keep as is and keep it consistent.

Reply:
It sure does, they just live in the autogenerated `kfp-server-api` library (they basically compile `protobuf` schemas into `python` classes/exceptions that live in that library), I've just updated all the `client` methods with the appropriate return type annotations now, we'll see what `kfp` says during the review! :)